docs/conf.py

```diff
@@ -86,6 +87,9 @@
 nitpicky = True
 nitpick_ignore = []
 
+# Force --colored default value to be yes.
+os.environ["FORCE_COLOR"] = "yes"
```

Comment:
can you put the command to the right-hand side here?

Reply:
Sorry I do not understand what you mean.

Reply:
```python
os.environ["FORCE_COLOR"] = "yes"   # ensure by default we documented colored
```

Reply:
I see now (you wrote command instead of comment).

Reply:
my bad :laughing: